Signal/src/ViewControllers/SelectRecipientViewController.m

```diff
@@ -442,9 +481,17 @@ - (void)updateTableContents
         [self.phoneNumberTextField autoPinEdge:ALEdgeLeft toEdge:ALEdgeRight ofView:phoneNumberLabel withOffset:0];
         [self.phoneNumberTextField autoVCenterInSuperview];
 
+        // Example row.
+        UIView *examplePhoneNumberRow = [self createRowWithHeight:examplePhoneNumberRowHeight
+                                                      previousRow:phoneNumberRow
+                                                        superview:cell.contentView];
+        [examplePhoneNumberRow addSubview:self.examplePhoneNumberLabel];
+        [self.examplePhoneNumberLabel autoVCenterInSuperview];
+        [self.examplePhoneNumberLabel autoPinEdgeToSuperviewEdge:ALEdgeRight withInset:20.f];
```

Comment:
Sorry I just saw this. Interesting, I've usually taken a different approach. I'll think this over!